python/lsst/meas/algorithms/matcherSourceSelector.py

```diff
@@ -119,16 +135,20 @@ def _hasCentroid(self, source):
         return np.all(np.isfinite(centroid)) and not source.getCentroidFlag()
 
     def _goodSN_vector(self, sourceCat):
-        """Return True for each source that has Signal/Noise > config.minSnr."""
+        """Return True for each source that has Signal/Noise > config.minSnr.
+        """
         if self.config.minSnr <= 0:
             return True
```

Comment:
This is a pre-existing issue, but please update the docs to say that it returns a scalar True if config.minSnr <= 0.

I'm guessing that works because numpy can perform mathematical operations on vectors with scalars. If there's any doubt then it's better to return [True]*len(sourceCat). But as long as this branch is tested it probably suffices to document what's going on.